guvnor-asset-mgmt/guvnor-asset-mgmt-client/src/test/java/org/guvnor/asset/management/client/editors/repository/wizard/CreateRepositoryWizardTest.java

```diff
@@ -293,6 +285,69 @@ public void testManagedRepositoryMultiCompletedTest() {
         WizardTestUtils.assertWizardComplete( true, createRepositoryWizard );
     }
 
+    @Test
+    public void testCompletionStatusSuccess() {
+        setupNameAndOrgUnitMocks(true, ORGANIZATIONAL_UNIT);
+
+        createRepositoryWizard.start();
+
+        infoPage.onNameChange();
+        infoPage.onOUChange();
+
+        verify(view, times(2)).setCompletionStatus(false);
```

Comment:
On line 292 is first call of setCompletionStatus(false) and setPageCompletionState(0, false). Because at start of wizard are both name and org.unit unfilled.

On line 294 is second call of setCompletionStatus(false) and setPageCompletionState(0, false). Because name is now filled, but not org.unit. Wizard is not completed.

On line 295 is wizard completed and call of setCompletionStatus(true) and setPageCompletionState(0, true) is executed here.